src/Cache/Engine/FileEngine.php

```diff
@@ -103,17 +103,6 @@ public function init(array $config = [])
     }
 
     /**
-     * Garbage collection. Permanently remove all expired and deleted data
-     *
-     * @param int|null $expires [optional] An expires timestamp, invalidating all data before.
-     * @return bool True if garbage collection was successful, false on failure
-     */
-    public function gc($expires = null)
```

Comment:
Reminds me of this by @AD7six:

http://ad7six.com/blog/2015/01/07/file-caching-preventing-running-out-of-space/